docs/supported_apis.md

```diff
@@ -1,3 +1,9 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="NVIDIA CUDA APIs supported by the HIPIFY tool">

```

Comment:
`tool`-> `tools`, which are `hipify-clang` and `hipify-perl` for now.

Reply:
I changed this to HIPIFY, rather than HIPIFY tools. Let me know if you think more is needed.